src/scripts/seqbox_utils.py

```diff
@@ -1186,7 +1221,7 @@ def check_extraction_fields(extraction_info):
          print(f'extraction_from column must be one of {allowed_extraction_from}, it is not for \n{extraction_info}\n. '
                f'Exiting.')
          sys.exit(1)
-    if extraction_info['nucleic_acid_concentration'].strip() == '':
+    if np.isnan(extraction_info['nucleic_acid_concentration']) is True:
```

Comment:
this is just the same as `if np.isnan(extraction_info['nucleic_acid_concentration']):`, which you used above? Be consistent.